src/js/modules/infragistics.ui.videoplayer.js

```diff
@@ -2501,7 +2508,9 @@
 			this._removeVideoProperty(video, "autoplay");
 			this._removeVideoProperty(video, "preload");
 			this._removeVideoProperty(video, "loop");
-			this._removeVideoProperty(video, "poster");
+			if (this._oldPoster !== "") {
```

Comment:
The `poster` attribute of the video element becomes `undefined` if it is originally not set. Which in turn reesults in another error. So we take no actions unless it was explicitly set.